static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -48,11 +49,24 @@ export default function FeedbackActions({
 }
 
 function LargeWidth({feedbackItem}: {feedbackItem: FeedbackIssue}) {
-  const {isResolved, onResolveClick, isSpam, onSpamClick, hasSeen, onMarkAsReadClick} =
-    useFeedbackActions({feedbackItem});
+  const {
+    hasDelete,
+    onDelete,
+    isResolved,
+    onResolveClick,
+    isSpam,
+    onSpamClick,
+    hasSeen,
+    onMarkAsReadClick,
+  } = useFeedbackActions({feedbackItem});
 
   return (
     <Fragment>
+      {hasDelete && (
+        <Button size="xs" onClick={() => openDeleteModal(onDelete)} priority="danger">
```

Comment:
Where does `openDeleteModal` come from here? I don't see it imported

Reply:
line 188!